tasks/release.rake

```diff
@@ -1,29 +1,13 @@
 # frozen_string_literal: true
 
 namespace :release do
-  desc 'Create a new prerelease, optionally publishing to rubygems.org'
-  task :pre, :publish do |_t, args|
-    cmd = 'gem bump --push --version pre'.dup
-    cmd.concat(' --tag --release') if args.publish == 'true'
-
-    sh cmd
-  end
-
-  desc 'Create a new release, dropping the current prerelease version, publish to rubygems.org'
-  task :none do
-    sh 'gem bump --tag --release --version release'
-    sh 'gem bump --push --version pre'
-  end
-
-  desc 'Create a new release with a minor version bump, publish to rubygems.org'
-  task :minor do
-    sh 'gem bump --tag --release --version minor'
-    sh 'gem bump --push --version pre'
+  desc 'publish the gem to rubygems'
+  task :publish do
+    sh 'gem release'
```

Comment:
I was hoping to find something to avoid using the command line with `sh`, but there doesn't seem to be much built-in. Something like this might be worth looking at: https://github.com/postmodern/rubygems-tasks I'm not convinced it's any better than the `sh` steps, though.

Reply:
I was partly considering just removing these tasks outright, since both the `build` and `publish` tasks only call `gem release` and `gem build` respectively; I leaned towards leaving the tasks in case we wanted to have them do more later (that way we wouldn't have to modify the CI workflow), but I could just as easily have the CI workflow directly just call `gem release`.

Reply:
I would prefer to have the CI workflow issue the `gem release` and `gem build` actions directly for now rather than add another layer of rake tasks. Or, if you really want to keep a rake task, have the `release` task do both the build and release. I don't see anything actually using the `build` task. What am I missing?